### PR TITLE
adds in show information and transcript for 015

### PR DESCRIPTION
**Pull request by nellshamrell**
Thank you again!

I will update the reddit link with the one where this week's issue of TWiR is being discussed once I post it later today.

I did not include timestamps for the show notes because I felt the show is short enough that it's not needed, but I am happy to add those in if you feel they are needed.

Signed-off-by: Nell Shamrell <nellshamrell@gmail.com>

### CODE REVIEW

File: _transcripts/015-transcription.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+
+Something that has encouraged me as much of the world remains in lockdown is seeing how many Rust meetups are still being held online, it is awesome that we can still connect with each other virtually. Many of these meetups are also posting their recordings on Youtube and it’s a great way to see content from around the world. This week’s newsletter features a video of the Rust and C++ Cardiff Virtual meetup in the UK. It features four talks on Rust and C++ as well as some lightning talks. If you’ve been missing your local Rust meetup, check out what’s available online, there’s a lot.
+Finally, I’ve also personally really been enjoying Rust developers streaming their development through YouTube, Twitch, and other services. If you are a newer Rust developer or an advanced developer who just wants to brush up on the fundamentals, you’ll like the called “Jonathan teaches Jason Rust.” You can follow right along as Jonathan Turner - a prominent Rust community member -  takes Jason Turner - A C++ expert - through the basics of Rust development.
```
Comment (ericseppanen) [inline _transcripts/015-transcription.md:22]: If this is a paragraph break, there should be an extra newline here.

File: _transcripts/015-transcription.md [hunk 2]
```diff
@@ -0,0 +1,50 @@
+Hello fellow Rustacean!
```
Comment (ericseppanen) [inline _transcripts/015-transcription.md:1]:
```suggestion
Hello fellow Rustaceans!
```

File: _transcripts/015-transcription.md (hunk 1 above)
Comment (ericseppanen) [inline _transcripts/015-transcription.md:22]: These dashes should probably be em-dash characters.

You can safely paste the unicode "—" character directly into markdown source.  That's what I've been doing, anyway.

**Comment by ericseppanen** [comment]
The transcript.md file should have the same name as the episode.md file.

The transcript.md file also needs the "front matter" at the top; as far as I know only the `title` and `file` fields are necessary.  Those are the only two I've been including, anyway.

**Comment by jonhoo** [comment]
Nice! Thanks for putting this together! I've recently pushed a fix to CI, so if you can merge `master`, you should get CI here too.

Two things we'll want to fix before merging:
 - You don't need the `layout` parts in the meta part of the Markdown any more. The [template](https://raw.githubusercontent.com/rustacean-station/rustacean-station.org/master/000-template.md) (`000-template.md` in the root) should have the most up-to-date structure.
 -  The transcript needs to have the same file name as the episode file (so `015-twir-338.md`). I should probably update CI to check this too.

**Comment by jonhoo** [comment]
Oh, yeah, and the front matter in the transcript file as @ericseppanen pointed out. I'll try to add that to CI too.

File: _transcripts/015-transcription.md (hunk 2 above)
Comment (ericseppanen) [inline _transcripts/015-transcription.md:1]:
```suggestion
__Jon Gjengset__: Hello fellow Rustacean!
```

File: _transcripts/015-transcription.md [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+
+Ben and I always harp on how Rustacean Station is a community podcast; that even though we are the ones who host episodes most frequently, we are always looking for other people with something to say in the Rust world. And to our delight, Nell Shamrell Harrington, the lead editor of This Week in Rust, got in touch with us the other day asking if Rustacean Station would be a good place to host a weekly This Week in Rust podcast! We jumped at the chance, and so without further ado, here is the very first episode.
+
+Hello everyone! This is the first episode of the This Week in Rust podcast. I’m Nell Shamrell-Harrington, I’m lead editor of This Week in Rust and an engineer on the Rust team at Mozilla.
```
Comment (ericseppanen) [inline _transcripts/015-transcription.md:5]: Is this 
```suggestion
__Nell Shamrell-Harrington__: Hello everyone! This is the first episode of the This Week in Rust podcast. I’m Nell Shamrell-Harrington, I’m lead editor of This Week in Rust and an engineer on the Rust team at Mozilla.
```

**Comment by ericseppanen** [comment]
I've been previewing my transcriptions by running jekyll in docker (mostly because I couldn't find a trustworthy debian package). It's an easy way to make sure all the metadata is there and check that the links, formatting, etc. came out right.

If others would find that workflow useful, I could try to make it portable and post a PR.

**Comment by ericseppanen** [comment]
Also, I've been word-wrapping transcripts at 80 columns.  It's not really necessary, but reviews and git diffs are a bit more readable that way.

I probably owe a style guide or something, with some of these tips.  You're the first person other than me to post a transcript, so thanks for blazing that trail. :)

**Comment by jonhoo** [comment]
If you merge with master, you should now get CI for transcripts too, which should catch all the errors above :)

**Comment by ericseppanen** [comment]
> If you merge with master, you should now get CI for transcripts too, which should catch all the errors above :)

Does that apply to PRs too?  Or does the check only get done when it's merged?

**Comment by jonhoo** [comment]
Should check PRs too. The only reason this one isn't tested is because I only fixed it on master just now :p Merging should make all the CI checking run.

**Comment by nellshamrell** [comment]
On it!

File: _episodes/015-twir-338.md [hunk 4]
```diff
@@ -0,0 +1,49 @@
+---
+layout: post
```
Comment (jonhoo) [inline _episodes/015-twir-338.md:2]: This should also no longer be necessary.

File: _transcripts/015-twir-338.md [hunk 5]
```diff
@@ -0,0 +1,56 @@
...
+
+Finally, there are several upcoming events in the Rust community. This week and next there are a lot of online meetups from across the world that are open to all. This includes meetups in Vancouver BC, Turin IT, Dallas TX, Berlin, and Montreal. There is also a meetup in Durham, NC, called the Triangle Rustaceans. Meetups are a great way to not only learn Rust, but also connect with the fantastic community that has made Rust so successful. And because many of these meetups are now online, you can connect with Rust communities from around the world.
+
+That’s all for this week’s podcast - there are a lot more items in this week’s newsletter, make sure to read it to keep up with all the awesome things going on in the Rust world.
+
+This Week in Rust is edited by myself, Andre Bogus, and Colton Donnelly. This week’s newsletter’s contributors include, using their GitHub usernames, mrcosta, nickwilcox, pksunkar, and zoni! Thank you so much to everyone who contributes to This Week in Rust!
+
+Thank you also to the Rustacean station for hosting this podcast! I will be back next week bringing you more of the wonderful news from the wonder world of Rust. Take care everyone.
+
+Show Notes
```
Comment (jonhoo) [inline _transcripts/015-twir-338.md:44]: The show notes down here can probably be removed, since they are already in the main show notes :)

Comment (jonhoo) [reply _transcripts/015-twir-338.md:44]: And if any of it isn't, then probably best to fold them into the show notes. The transcript should ideally be _just_ transcript.

**Comment by jonhoo** [comment]
I think we're pretty close now! Just the last two nits, and then I think we're good :tada:

File: _transcripts/015-twir-338.md (hunk 5 above)
Comment (nellshamrell) [reply _transcripts/015-twir-338.md:44]: got it!

**Pull request merged**